modules/enabling-etcd-encryption.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * authentication/encrypting-etcd.adoc
+
+[id="enabling-etcd_{context}"]
+= Enabling etcd encryption
+
+You can enable etcd encryption to encrypt sensitive resources in your cluster.
+
+// [WARNING]
+// ====
+// Do not back up etcd until the encryption process is complete.
+// ====
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Modify the API server object:
++
+----
+$ oc edit apiserver
+----
+
+. Set the `encryption` field type to `aescbc`:
++
+[source,yaml]
+----
+spec:
+  encryption:
+    type: aescbc
+----
```

Comment:
If you have the information handy, I might add a sentence about why `aescbc` and a statement about whether or not there are other encryption options.

Reply:
Good suggestion. Will add this in my next PR as I'll want that reviewed (and want to merge this PR now).